CustomDcrAndDceArmTemplates/CustomDCE.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Rakshith - Can you create a top level folder named 'Azure-ARM-templates' and put your new folder underneath it ? Also can you rename your folder to say 'Prometheus-RemoteWrite-DCR-artifacts' ?